core/src/main/java/bisq/core/app/WalletAppSetup.java

```diff
@@ -90,8 +88,8 @@ void init(@Nullable Consumer<String> chainFileLockedExceptionHandler,
               Runnable walletPasswordHandler,
               Runnable downloadCompleteHandler,
               Runnable walletInitializedHandler) {
-        log.info("Initialize WalletAppSetup with BitcoinJ version {} and LibDohJ version {} with hash of BitcoinJ commit {}",
-                VersionMessage.BITCOINJ_VERSION, Version.VERSION, Version.BITCOINJ_VERSION);
+        log.info("Initialize WalletAppSetup with BitcoinJ version and hash of BitcoinJ commit {}",
```

Comment:
I think here is missing the {} -> BitcoinJ version {} and

Reply:
Beside that ACK (just tested).

Reply:
Done. just force pushed the fix.